src/Microsoft.Extensions.Configuration.AzureAppConfiguration/IConfigurationClientManager.cs

```diff
@@ -10,7 +10,7 @@ namespace Microsoft.Extensions.Configuration.AzureAppConfiguration
 {
     internal interface IConfigurationClientManager
     {
-        IEnumerable<ConfigurationClient> GetAvailableClients(DateTimeOffset time);
+        IEnumerable<ConfigurationClient> GetAvailableClients(DateTimeOffset time, bool ignoreBackoff = false);
```

Comment:
It doesn't make sense to pass the time and the ignore backoff parameter.

Perhaps a separate method is warranted. GetClients, which returns all clients.

Reply:
How about `GetAllClients()` to better differentiate from `GetAvailableClients`?